js/repl/ReplOptions.js

```diff
@@ -450,7 +450,7 @@ const styles = {
     }
   }),
   inputCheckboxLeft: css({
-    marginRight: '0.5rem'
+    margin: '0 0.5rem 0 0 !important' // Clashes with default input[type="checkbox"] styles in main.css
```

Comment:
I don't think more-specific would be any less hacky in this case, since nested selectors are slow and kind of gross. The problem is that we set (gross) global styles for checkboxes and radio. I think _that's_ where we should fix this TBH, but I'm new here and didn't want to make too many splashes.